mne/channels/montage.py

```diff
@@ -312,7 +312,8 @@ class DigMontage(object):
     Parameters
     ----------
     hsp : array, shape (n_points, 3)
-        The positions of the channels in 3d.
+        The positions of the headshape points in 3d.
+        These points are in the acquisition digitizer space.
```

Comment:
Which space is that? Do you mean `FIFFV_MNE_COORD_DIGITIZER` / `FIFFV_COORD_ISOTRAK`, because that's what it sounds like. Or do you mean head coords (`FIFFV_COORD_HEAD`)? I don't see any code to transform to `FIFFV_COORD_ISOTRAK`, so I assume you mean `FIFFV_COORD_HEAD`.